### ISSUE TITLE
tools.cherrypy.org doesn't work anymore

### ISSUE THREAD

**Issue by stenci** [cherrypy/cherrypy#1590]
I keep finding broken links to tools.cherrypy.org and other cherrypy related domains on Google and Stackoverflow search results. [This](https://groups.google.com/forum/#!searchin/cherrypy-users/AuthenticationAndAccessRestrictions%7Csort:relevance/cherrypy-users/htrsx0bvpOA/VYfUfY0CRVMJ) and [this](http://stackoverflow.com/questions/12361137/cherrypy-session-authentication) are just the last two quick examples that I found.

Are all those CherryPy resources lost forever?

**Comment by webknjaz** [comment]
I think yes, let's ask @jaraco and @Lawouach

**Comment by webknjaz** [comment]
@stenci I think, you might be interested in looking historical snapshot of the resource: https://web-beta.archive.org/web/20160723085955/http://tools.cherrypy.org:80/wiki/TitleIndex
and Google cache: http://webcache.googleusercontent.com/search?q=cache%3A%2F%2Ftools.cherrypy.org%2Fwiki%2FTitleIndex&oq=cache%3A%2F%2Ftools.cherrypy.org%2Fwiki%2FTitleIndex&aqs=chrome..69i57j69i58.14654j0j4&sourceid=chrome&ie=UTF-8

**Comment by stenci** [comment]
The links I provided are just an example. I hoped that there would be an easy fix like for my previous [issue](https://github.com/cherrypy/cherrypy/issues/1571).

I started using CheryPy two years ago and I was able to find documentation, material, posts and examples. I tried again now, and it's impossible. 

I decided to give up. I am switching to Pyramid for my next project because searching for CherryPy support or documentation has become too difficult compared to two years ago.
